core/app/models/spree/adjustment.rb

```diff
@@ -56,6 +56,14 @@ class Adjustment < Spree::Base
     extend DisplayMoney
     money_methods :amount
 
+    def self.adjusted_orders(excluded_orders: [])
```

Comment:
I have a couple of questions about this class method/scope:

1. Maybe a different name could help understand what this method does: what about a simpler `in_completed_orders()`? I think that maybe with `adjusted_orders()` could not be immediately clear that it's just searching into completed orders.
2. I'm not sure that `eligible` should be part of this method. Chaining it from outside adds a bit of duplication but makes the code more flexible IMO, what do you think?

Reply:
I agree on both points.
At first I had used a much longer name but it was quite hard to read, `in_completed_orders` strikes a good balance.